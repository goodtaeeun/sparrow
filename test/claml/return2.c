void fn() { return 0; }

int main() {
  fn();
  return 0;
}
