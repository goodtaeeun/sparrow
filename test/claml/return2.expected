/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "return2.c"
void fn(void) {

  { return; }
}
#line 3 "return2.c"
int main(void) {

  {
    {
#line 4
      fn();
    }
#line 5
    return (0);
  }
}
